### DISCUSSION TITLE
win10下liteide下载以后执行文件可以直接拖到面板上面

### DISCUSSION THREAD

**Discussion by widon1104** [linuxdeepin/developer-center#2969] [Features Request & Ideas | 特性请求 & 头脑风暴]
## 缺陷描述

win10下sourceforge下载 liteidex37.1-1.windows-qt5.9.5.zip
解压以后在liteide/bin下面有liteide.exe文件，这个文件可以直接拖到面板上面去
而在deepin下面下载了安装包以后需要手动编写Desktop文件，这明显非常麻烦

有人说使用终端命令行安装liteide就可以，但是终端命令行安装软件有一个不好的地方就是这种方式有很多默认的配置在里面，而这些配置都不是我想要的，有时候还是需要下载安装包的方式安装软件

## 期望行为

执行文件直接拖就可以了，其实无非就是1）找到可执行文件 2）找到可执行文件的图标，满足这2项最基本的就行了，其他的再优化

### 发行版以及版本

deepin15.11

**Comment by XMuli** [comment]
The biggest advantage of linux is the use of the terminal to run the installation, the WIN advantage is the use of the GUI; of course, you are more than welcome to submit a PR, which is even better than issuse (bug).

Especially if you think it's very simple, very easy to do, but you want to entrust it to someone else, if you don't like slow progress, you can try to write your own code submission; your code will be merged first

**Comment by widon1104** [comment]
我随便说几点
1）我最近在公司使用mac，mac也有类似的功能，把程序加入面板是非常方便的
2）这个问题不应该关，因为问题没有解决，我觉得deepin应该开发这种功能，不要再让用户自己写desktop文件，能自动生成就自动生成
3）我没说这功能简单，但是国产系统要做好的话应该要加上这个功能，微软苹果系统都已经加上了类似的功能，还有中国人的话建议还是说中文比较好，估计没有老外看
